build/patches/Restore-Search-Ready-Omnibox-flag.patch

```diff
@@ -149,16 +157,15 @@ diff --git a/chrome/browser/ui/android/omnibox/java/src/org/chromium/chrome/brow
          registerSuggestionProcessor(
                  new AnswerSuggestionProcessor(context, host, textProvider, imageFetcherSupplier));
          registerSuggestionProcessor(
-@@ -211,6 +213,10 @@ class DropdownItemViewInfoListBuilder {
- 
+@@ -210,6 +213,9 @@ class DropdownItemViewInfoListBuilder {
      /** Signals that native initialization has completed. */
      void onNativeInitialized() {
+         mHeaderProcessor.onNativeInitialized();
 +        if (ChromeFeatureList.isEnabled(ChromeFeatureList.SEARCH_READY_OMNIBOX) == false) {
```

Comment:
I am not sure, but in my opinion it is more correct before `mHeaderProcessor.onNativeInitialized();`

Reply:
`mHeaderProcessor` does not interact with `mPriorityOrderedSuggestionProcessors`, I'll leave it here because we remove one processor before the for loop that initializes all of them.

Reply:
> I'll leave it here because we remove one processor before the for loop that initializes all of them.

sure, but since registration is first and we insert deletion code there just for convenience, theoretically we should enter our code first of any upstream activity. 
but in any case it is the same since that call seems to be inserted only for tests

Reply:
If `mHeaderProcessor.onNativeInitialized()` does any business with the suggestion processors then we should move the removal there, but since it only initializes the header processor this is not needed.